test-project/Assets/Config/GdkToolsConfiguration.json

```diff
@@ -0,0 +1 @@
+{"schema_standard_lib_directory":"build/dependencies/schema/standard_library","schema_source_directories":["Assets/.Schema"],"codegen_output_directory":"Assets/Generated/Source"}
```

Comment:
nit: can you put fields in separate lines (will help with reviewing when parts change later)

Reply:
I can manually make the change - but any auto-generated JSON will have this problem 😢

Reply:
`JsonUtility` has pretty-printing capabilities - it should also make it easier to parse into/out of a `[Serializable]` struct rather than a dictionary. It may be worth giving it a try if you haven't already :)

Reply:
The reason I didn't use `JsonUtility` is that it doesn't support lists/arrays out of the box and you have to implement wrappers to work around it.

We also use `MiniJSON` elsewhere so was following precedent. I think @jared-improbable has some oppositions to `JsonUtility`

Reply:
Lists/arrays work, dictionaries are the ones that are difficult.

```cs
        [Serializable]
        public struct SomeStruct
        {
            public string schema_standard_lib_directory;
            public string[] schema_source_directories;
            public string codegen_output_directory;
        }

        [Test]
        public void JSONUtilityTest()
        {
            var myStruct = JsonUtility.FromJson<SomeStruct>(@"{ ""schema_standard_lib_directory"":""build/dependencies/schema/standard_library"", ""schema_source_directories"":[""Assets/.Schema""],""codegen_output_directory"":""Assets/Generated/Source""}");

            Debug.Log(JsonUtility.ToJson(myStruct, true));
        }
```

Output:
```json
{
    "schema_standard_lib_directory": "build/dependencies/schema/standard_library",
    "schema_source_directories": [
        "Assets/.Schema"
    ],
    "codegen_output_directory": "Assets/Generated/Source"
}
```

Reply:
It's Sunday so I'll allow myself a meme

![internet-lies](https://i0.kym-cdn.com/photos/images/newsfeed/000/428/075/30a.jpeg)

Reply:
Adjusted to use `JSONUtility`, PTAL :)